.github/workflows/buildkit.yml

```diff
@@ -48,6 +48,10 @@ jobs:
           worker: oci
           tags: nydus
           typ: integration
+        - pkg: ./frontend/dockerfile
+          worker: containerd
+          tags: nydus
+          typ: integration
```

Comment:
dockerfile frontend tests are executed in frontend workflow. Would just need to add:

```
      tags: |
        ''
        nydus
```

in https://github.com/moby/buildkit/blob/61e6f28bcf3e19f4f4f5d90761624d7478b42da4/.github/workflows/frontend.yml#L29-L37

Reply:
@crazy-max Thanks for the reply, fixed. But it seems introduced many jobs (17 nydus related in total), is there a better way to limit only run the test `frontend / test / run (containerd, ./frontend/dockerfile, integration, nydus)`?

Reply:
Right that seems unnecessary to have that much. Maybe we could just add an include against a limited set of workers:

```yaml
jobs:
  test:
    uses: ./.github/workflows/.test.yml
    with:
      cache_scope: frontend-integration-tests
      pkgs: ./frontend/dockerfile
      kinds: |
        integration
        dockerfile
      codecov_flags: dockerfile-frontend
      includes: |
        - tags: nydus
          pkg: ./frontend/dockerfile
          kinds: |
            integration
            dockerfile
          worker: |
            containerd
            oci
```